hbase-client/src/main/java/org/apache/hadoop/hbase/filter/ParseFilter.java

```diff
@@ -254,14 +254,9 @@ public Filter parseSimpleFilterExpression (byte [] filterStringAsByteArray)
       Class<?>[] argTypes = new Class [] {ArrayList.class};
       Method m = c.getDeclaredMethod("createFilterFromArguments", argTypes);
       return (Filter) m.invoke(null,filterArguments);
-    } catch (ClassNotFoundException e) {
-      e.printStackTrace();
-    } catch (NoSuchMethodException e) {
-      e.printStackTrace();
-    } catch (IllegalAccessException e) {
-      e.printStackTrace();
-    } catch (InvocationTargetException e) {
-      e.printStackTrace();
+    } catch (ClassNotFoundException | NoSuchMethodException | InvocationTargetException |
+        IllegalAccessException e) {
+      LOG.error(e.getMessage(), e);
```

Comment:
Similar story here... The original exception is not passed into the exception which is thrown further down so a log message is justified - but at most DEBUG or INFO I think...

Reply:
I would actually argue that the `throw` below should be moved here, `e` should be provided as the cause of the thrown exception and the log message should be removed, based on the reasoning explained in my other comment.